cli.sh

```diff
@@ -1030,12 +1030,11 @@ list_shared_hook_repos() {
             echo "  - None"
         else
             for LIST_ITEM in $(git config --global --get githooks.shared); do
-                NORMALIZED_NAME=$(echo "$LIST_ITEM" |
-                    sed -E "s#.*[:/](.+/.+)\\.git#\\1#" |
-                    sed -E "s/[^a-zA-Z0-9]/_/g")
 
-                if [ -d ~/.githooks/shared/"$NORMALIZED_NAME"/.git ]; then
-                    if [ "$(cd ~/.githooks/shared/"$NORMALIZED_NAME" && git config --get remote.origin.url)" = "$LIST_ITEM" ]; then
+                set_shared_root "$LIST_ITEM"
+                echo "$SHARED_ROOT"
+                if [ -d "$SHARED_ROOT/.git" ]; then
+                    if [ "$(cd "$SHARED_ROOT/.git" && git config --get remote.origin.url)" = "$LIST_ITEM" ]; then
```

Comment:
Slight change from the old code, we used to `cd` into repo root, not the `repo/.git` folder. Maybe it doesn't make a difference, but if it works, would be nice to go back to that one please.